features/SideNavigation.feature

```diff
@@ -0,0 +1,165 @@
+        Given the user is on the docs portal
+        Then they see the following side navbar item types
+
+            | types                                                    |
+            | Core Category Section with navigation arrow              |
+            | Core Category Section with icon only                     |
+            | Navigation item with an arrow down                       |
+            | Article item                                             |
+            | Core category item at the top of the navbar with an icon |
+
+    Scenario: The Core Category Section with navigation arrow takes you to another level of navigation items and opens up a page
+        Given the side navbar is in the starting state
+        When the user clicks on any part of a " Core Category Section with navigation arrow"
+        Then they see that in the nav bar a "Core category item at the top of the navbar with an icon" shows up
+        And the sub-category items are there under it
+        And the home page opens up for the selected core category
+
+    Scenario: Core Category Section with icon only
+        Given the side navbar is in the starting state
+        Then SaaSquatch Product News are displayed as the last menu item without an arrow
+        When the user clicks on this item
+        Then a home page for Product News opens up
+        And there are no sub-categories for this item type
+
+    Scenario: The navigation item with an arrow down opens a dropdown menu
+        Given the user selected a core category
+        When the user clicks on a "navigation item with an arrow down"
+        Then the list of articles are shown under the item
+        And there could be further dropdown items between the articles
+        But the page is not changing
+
+    Scenario: The navigation item with an arrow up closes a dropdown menu
+        Given the user selected a "navigation item with an arrow down"
+        When the user clicks on a "navigation item with an arrow up"
+        Then they see the list of articles are closed
+        And the arrow changes back to down
+
+    Scenario: The article item in the menu opens up an article
+        Given the user opened up a "navigation item with an arrow down"
+        When the user clicks on the "article" item
+        Then the article opens up in the central section of the page
+
+    Scenario: The core category item at the top of the navbar with an icon stays there until you are at that category and used to navigate back to the home page of this category
+        Given the user navigates within a core category
+        Then they see that the "Core category item at the top of the navbar with an icon" item is visible all the time
+        When they click on this item from anywhere withing the category
+        Then the home page opens up for this core category
+        And it changes to a selected state until the user is on the home page
```

Comment:
Not sure I follow this, if it changes to a selected state until the user is on the home page - does that mean its unselected when they get there?